hera_cal/frf.py

```diff
@@ -16,11 +16,85 @@
 from pyuvdata import UVData, UVFlag
 import argparse
 from . import io
+from . import vis_clean
 import warnings
+import astropy.constants as const
 
 
-def timeavg_waterfall(data, Navg, flags=None, nsamples=None, wgt_by_nsample=True, 
-                      wgt_by_favg_nsample=False, rephase=False, lsts=None, freqs=None, 
+SPEED_OF_LIGHT = const.c.si.value
+SDAY_KSEC = 86163.93 / 1000.
+
+
+def sky_frates(uvd, keys=None, frate_standoff=0.0, frate_width_multiplier=1.0, min_frate_half_width=0.025):
+    """Automatically compute sky fringe-rate ranges based on baselines and telescope location.
+
+    Parameters
+    ----------
+    uvd: UVData object
+        uvdata object of data to compute sky-frate limits for.
+    keys: list of antpairpol tuples, optional
+        list of antpairpols to generate sky fringe-rate centers and widths for.
+        Default is None -> use all keys in self.data.
+    frate_standoff: float, optional
+        Additional fringe-rate standoff in mHz to add to Omega_E b_{EW} nu/c for fringe-rate inpainting.
```

Comment:
additional fringe-rate to add to computed fringe-rate bounds [mHz]

mathematic terms are somewhat confusing without definition, and no need to specify fringe-rate inpainting as this func should be general anyways

Reply:
fixed and changed math to "to add to analytic calculation of fringe-rate bounds for emission on the sky."

Reply:
for some reason I still see math and not the comment you said above--is my browser page stale or something, or did you forget to push this up?

Reply:
I think I changed this in one place but not the other. I've fixed it here too.